packages/lesswrong/components/posts/PostsTimeBlock.tsx

```diff
@@ -36,8 +48,14 @@ const styles = (theme: ThemeType): JssStyles => ({
     marginTop: 6,
   },
   noPosts: {
-    marginLeft: "23px",
+    marginLeft: [isEAForum ? 0 : "23px"],
```

Comment:
@agnesstenlund you don't need those square brackets

```suggestion
    marginLeft: isEAForum ? 0 : 23,
```

(Also whoever wrote that file originally didn't realize that raw numbers get interpreted as pixels.

Reply:
thank you 🤗